test/unit/org/apache/cassandra/tools/nodetool/SetGetColumnIndexSizeTest.java

```diff
@@ -66,20 +67,20 @@ public void testZero()
     @Test
     public void testNegative()
     {
-        assertSetInvalidColumnIndexSize("-7", "column_index_size_in_kb must be positive value < 2097152, but was -7", 2);
+        assertSetInvalidColumnIndexSize("-7", "DataStorage value must be positive, but was -7", 2);
```

Comment:
nit: Might be good to not expose `DataStorage` to the user and keep `column_index_size` in the error message.

Reply:
We do the negative check in the DataStorage. I changed the message as per previous discussion:
`Invalid data storage: value must be positive, but was`